script.js

```diff
@@ -0,0 +1,42 @@
+const inputToDo = document.querySelector('.inputToDo');
+const buttonClick = document.querySelector('.inputButton');
```

Comment:
변수의 이름은 그 기능이 잘 드러나게...!
```suggestion
const newTodoItemAddButton = document.querySelector('.inputButton');
```